Source/SuperLinq/JoinType.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace SuperLinq;
+
+/// <summary>
+/// Values defining which type of join to use in join functions.
+/// </summary>
+public enum JoinType
+{
+	/// <summary>
+	/// Use nested loops over the left and right inputs to execute the join.
+	/// </summary>
+	Loop,
+
+	/// <summary>
+	/// Build a <see cref="ILookup{TKey, TElement}"/> for one side to execute the join.
+	/// </summary>
+	Hash,
+
+	/// <summary>
+	/// Sort both inputs and simultaneously enumerate both to execute the join.
```

Comment:
_reword_ As noted in remarks, it doesn't sort as implied here. It relies on it already be sorted.

Reply:
Good catch - Will update the docs